src/w3c/templates/sotd.js

```diff
@@ -7,7 +7,7 @@ ${
     conf.isPreview
       ? html`
   <details class="annoying-warning" open="">
-    <summary>This is a preview</summary>
+    <summary>This is a preview of PR #<a href="${conf.prUrl}">${conf.prNumber}</a></summary>
```

Comment:
We need to handle the case when `prUrl` and/or `prNumber` are missing. Otherwise, it'll render as:
``` html
<summary>This is a preview of PR #<a></a></summary>
```
Saying only "This is a preview" should be enough in that case.

Reply:
Same needs to be done in [w3c/templates/cgbg-sotd.js](https://github.com/w3c/respec/blob/develop/src/w3c/templates/cgbg-sotd.js) 🙂

Reply:
Let's use "pull request" instead of "PR".

Reply:
@yatri1609 See: https://github.com/w3c/respec/pull/1843#discussion_r222453719
Lets get this merged!

Reply:
> @yatri1609 See: [#1843 (comment)](https://github.com/w3c/respec/pull/1843#discussion_r222453719)
> Lets get this merged!

Yeah I'll add that part to it also.